packages/storage/system-test/storage.js

```diff
@@ -35,8 +35,7 @@ var Bucket = Storage.Bucket;
 var File = Storage.File;
 
 describe('storage', function() {
-  var IS_CI = env.projectId === 'long-door-651';
-  var TESTS_PREFIX = 'gcloud-test-';
+  var TESTS_PREFIX = 'gcloud-tests-';
```

Comment:
There are lingering buckets with the old prefix that I can't delete. I changed the prefix going forward... not sure if this is an ideal solution. https://console.cloud.google.com/storage/browser?project=long-door-651